src/style.css

```diff
@@ -0,0 +1,81 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+  font-style: normal;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+}
+
+.search-form {
+  margin-top: 20px;
+}
+
+.search_input {
+  font-family: inherit;
+  width: 100%;
+  padding: 0;
+  font-weight: 300;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  outline: none;
+  color: #3d4e61;
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
```

Comment:
Pay attention to CSS properties order
https://9elements.com/css-rule-order/

Reply:
I'm corrected my files css and html. Check my work please))

Reply:
> Pay attention to CSS properties order https://9elements.com/css-rule-order/

I'm corrected my files css and html. Check my work please))